webapp/channels/src/i18n/en_AU.json

```diff
@@ -2034,6 +2034,8 @@
   "admin.reporting.workspace_optimization.configuration.file_storage.cta": "Config file storage",
   "admin.reporting.workspace_optimization.configuration.file_storage.description": "Check your file storage settings to ensure your Mattermost workspace has access to the configured file storage.",
   "admin.reporting.workspace_optimization.configuration.file_storage.title": "File storage access is faulty.",
+  "admin.reporting.workspace_optimization.configuration.root_check.title": "Mattermost is running as root",
+  "admin.reporting.workspace_optimization.configuration.root_check.description":"Running Mattermost as root is not recommended. Please use a non-root user.",
```

Comment:
You don't have to add any translation besides the ones in `en.json`. The translation system will take care of them.